### DISCUSSION TITLE
Lookup Fields link to Lookup record

### DISCUSSION THREAD

**Discussion by russellandsusan** [rappen/FetchXMLBuilder#496] [Ideas]
Hi again mate,

I know that I can double click and open the record in the Results View and link to the record in D365 but it only pulls the top level table/entity record. Would it be possible to make the Results View link Lookup Fields to the record it is representing?

Rus.

**Comment by rappen** [comment]
Sorry, late answer...

For lookup-attribute you can "single click" then for open them!

![image](https://user-images.githubusercontent.com/2572253/128752881-f2d56362-0e75-4eff-a2d2-404dc1b4eab7.png)
